src/test/java/database/service/JsonDatabaseServiceTest.java

```diff
@@ -24,10 +24,10 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class JsonDatabaseServiceTest {
-    private final Student firstStudent = new Student("Mikhail Shashin", 5.0, List.of(0));
-    private final Student secondStudent = new Student("Nadezhda Shashina", 4.5, List.of(1));
-    private final Student thirdStudent = new Student("Nika Shashina", 5.0, List.of(1));
-    private final Student fourthStudent = new Student("Harry Potter", 5.0, List.of(0));
+    private final Student firstStudent = new Student("Mikhail Shashin", 5.0);
+    private final Student secondStudent = new Student("Nadezhda Shashina", 4.5);
+    private final Student thirdStudent = new Student("Nika Shashina", 5.0);
+    private final Student fourthStudent = new Student("Mikhail Shashin", 5.0);
```

Comment:
Please don't use real people's name in tests, you can use Harry Potter characters instead. Remove real names from all tests.